README.md

```diff
@@ -56,32 +64,32 @@ Where `<base-encoding-character>` is used according to the multibase table.
-base256emoji,      🚀,    base256 with custom alphabet using variable-sized-codepoints, draft
+encoding,          code,        description,                                                  status
+identity,          0x00 (NUL),  8-bit binary (encoder and decoder keeps data unmodified),     default
+base2,             0x30 (0),    binary (01010101),                                            candidate
+base8,             0x37 (7),    octal,                                                        draft
+base10,            0x39 (9),    decimal,                                                      draft
+base16,            0x66 (f),    hexadecimal,                                                  default
+base16upper,       0x46 (F),    hexadecimal,                                                  default
+base32hex,         0x76 (v),    rfc4648 case-insensitive - no padding - highest char,         candidate
+base32hexupper,    0x56 (V),    rfc4648 case-insensitive - no padding - highest char,         candidate
+base32hexpad,      0x74 (t),    rfc4648 case-insensitive - with padding,                      candidate
+base32hexpadupper, 0x54 (T),    rfc4648 case-insensitive - with padding,                      candidate
+base32,            0x62 (b),    rfc4648 case-insensitive - no padding,                        default
+base32upper,       0x42 (B),    rfc4648 case-insensitive - no padding,                        default
+base32pad,         0x63 (c),    rfc4648 case-insensitive - with padding,                      candidate
+base32padupper,    0x43 (C),    rfc4648 case-insensitive - with padding,                      candidate
+base32z,           0x68 (h),    z-base-32 (used by Tahoe-LAFS),                               draft
+base36,            0x6b (k),    base36 [0-9a-z] case-insensitive - no padding,                draft
+base36upper,       0x4b (K),    base36 [0-9a-z] case-insensitive - no padding,                draft
+base58btc,         0x7a (z),    base58 bitcoin,                                               default
+base58flickr,      0x5a (Z),    base58 flicker,                                               candidate
+base64,            0x6d (m),    rfc4648 no padding,                                           default
+base64pad,         0x4d (M),    rfc4648 with padding - MIME encoding,                         candidate
+base64url,         0x75 (u),    rfc4648 no padding,                                           default
+base64urlpad,      0x55 (U),    rfc4648 with padding,                                         default
+proquint,          0x70 (p),    PRO-QUINT https://arxiv.org/html/0901.4016,                   draft
+base256emoji,      0xe7 (🚀),    base256 with custom alphabet using variable-sized-codepoints, draft
 ```
 
 **NOTE:** Multibase-prefixes are encoding agnostic. "z" is "z", not 0x7a ("z" encoded as ASCII/UTF-8). For example, in UTF-32, "z" would be `[0x7a, 0x00, 0x00, 0x00]`.
```

Comment:
this note would need updating if we did this

Reply:
i'm not sure I understand-- I think the meaning of the note is that the UTF-8 input that generates the one-character prefix may be a useful guide but can't be relied upon, which is why I moved the UTF-8 into a second column and put it to the RIGHT instead of the LEFT :D